colcon.meta

```diff
@@ -14,5 +14,18 @@
                 "-DQT_PATH=/opt/qt515",
             ],
         },
+
+        "microxrcedds_agent": {
```

Comment:
`UAGENT_P2P_PROFILE` is ON by default, leading to a dependency on `microxrcedds_client`:
https://github.com/eProsima/Micro-XRCE-DDS-Agent/blob/master/CMakeLists.txt#L82

This may result in an unnecessary `microxrcedds_client` build on its SuperBuild process.

Reply:
Solved in 67c996dadc525e51203930f0ee3511cf08e916ba